Makefile

```diff
@@ -0,0 +1,5 @@
+SRCS := $(wildcard source/*.c)
```

Comment:
I dont think we want to have this in the root. makefile that is just for cmdline tool

Reply:
Also yotta_ignore file